dialogue-client-test-lib/src/main/java/com/palantir/dialogue/AbstractChannelTest.java

```diff
@@ -107,21 +103,21 @@ public void before() {
     @Test
     public void respectsBasePath_emptyBasePath() throws InterruptedException {
         channel = createChannel(server.url("").url(), MoreExecutors.newDirectExecutorService());
-        channel.createCall(endpoint, request).execute(observer);
+        channel.createCall(endpoint, request);
```

Comment:
this method should now be called "call" or "execute" or something

Reply:
done